DenisZackharov/teacherss
# Director can see list teachers

### Summary

None.

### How it works

https://user-images.githubusercontent.com/78980031/226123946-9cec72b8-60fb-45df-ad09-95230bb192f2.mov

### Test plan

List of steps to manually test introduced functionality:

1. Run server with bin/dev
2. Login with credentials: `director@example.com`, `123456`
3. Click on company name
4. Click on Staff
5. You can see list of teachers

# Checklist:


### Deploy notes

Notes regarding deployment the contained body of work.
These should note any db migrations, ENV variables, services, scripts, etc.
